libs/langchain/langchain/tools/render.py

```diff
@@ -7,7 +7,6 @@
 
 # For backwards compatibility
 from langchain_core.tools import (
-    ToolsRenderer,
```

Comment:
This btw is breaking crewai library (crewai 0.51.1) with the following error occuring: 

```md
from langchain.tools.render import render_text_description
  File "/foo/bar/buzz/venv/lib/python3.11/site-packages/langchain/tools/render.py", line 9, in <module>
    from langchain_core.tools import (
ImportError: cannot import name 'ToolsRenderer' from 'langchain_core.tools' (/Users/foo/bar/buzz/venv/lib/python3.11/site-packages/langchain_core/tools/__init__.py)
```